src/Command/AutomaticDeployCommand.php

```diff
@@ -122,6 +122,14 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     'pluginId' => $configuredPluginState->getId(),
                 ]);
                 $app->run($input, $output);
+            } else {
+                // Try to uninstall
+                $input = new ArrayInput([
+                    'command' => 'sd:plugins:uninstall',
+                    '--secure' => null,
```

Comment:
I think I have to check this first. I messed up something, I think this has to be:

`'secure' => true`